docs/zone-pivot-groups.yml

```diff
@@ -138,3 +138,11 @@ groups:
       title: Semantic Kernel
     - id: azure-openai-sdk
       title: Azure OpenAI SDK
+- id: azure-development-tool-set-two
```

Comment:
Can you colocate this with the other similar one? Also, should this be titled "Azure interface" instead?

Reply:
Moved and revised the title!